helmfile.d/charts/hnc/controller/templates/user-rbac.yaml

```
@@ -84,5 +84,10 @@ subjects:
   kind: User
   name: {{ . }}
 {{- end }}
+{{- range $.Values.rbac.user.serviceaccounts }}
```

Since we have the variable camel-cased in our templating, maybe keep it that way in the chart as well?
```suggestion
{{- range $.Values.rbac.user.serviceAccounts }}
```